ui/lib/core/addon/components/kv-object-editor.js

```diff
@@ -81,4 +84,16 @@ export default class KvObjectEditor extends Component {
       this.args.onKeyUp(event.target.value);
     }
   }
+  @action
+  validateKey(rowIndex, event) {
+    const { value } = event.target;
+    const keyHasWhitespace = new RegExp('\\s', 'g').test(value);
```

Comment:
We have a util you could use here https://github.com/hashicorp/vault/blob/main/ui/app/utils/validators.js#L31-L34

Reply:
That lives in the main app unfortunately. I thought about moving it to the core addon but given that it's a one liner I opted to go this route.

Reply:
Ah - I forgot that the models that use it also live in the main app 🙃